app/components/Puzzle/Context.jsx

```diff
@@ -315,7 +329,14 @@ const PuzzleContextProvider = ({ initialGrid, puzzleId, children }) => {
   };
 
   const prevDownClue = () => {
-    const [row, column] = activeCell;
+    const [activeRow, activeColumn] = activeCell;
+    let row = activeRow;
+    let column = activeColumn;
+    
+    if (grid[activeRow][activeColumn].clue.downClueNumber && !grid[activeRow][activeColumn].clue.isDownStart){
+      [row, column] = findClueStartCell(grid, grid[activeRow][activeColumn].clue.downClueNumber, "down");
+    }     
+    
```

Comment:
So @psim3141 these changes are checking if we are in a clue, just go to the start of that clue, vs. going to the previous clue (and vice versa for the next)? Could we do those checks in the "getPrevDownClueStart" and "getNextDownClueStart" functions or will they not work there?

Reply:
I think they would work there as well, should I modify the code?